### PR TITLE
ENH: Updation of ITK tag for python build

### PR DESCRIPTION
**Pull request by PranjalSahu**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,6 +43,6 @@
     keywords='ITK InsightToolkit',
     url=r'https://itk.org/',
     install_requires=[
-        r'itk>=5.2rc2'
+        r'itk>=5.3rc3'
```
Comment (dzenanz) [inline setup.py:46]: Maybe also update `version` on line 15.

Comment (PranjalSahu) [reply setup.py:46]: Yes, we will do it once everything is finalized before the conference.
We have to update the readme also.

Comment (PranjalSahu) [reply setup.py:46]: Are we removing Python 36, 37 version support?

Comment (dzenanz) [reply setup.py:46]: I think we are dropping 3.6, but keeping 3.7.

**Review by thewtex — APPROVED** [review]
:green_heart: :rocket:

**Pull request merged**